### ISSUE TITLE
Failed to install Ubermag using conda on Ubuntu 22.02

### ISSUE THREAD

**Issue by KylinGuo** [ubermag/ubermag#152]
I am using the following commands to install ubermag on Ubuntu 22.02. 
```
$ conda create -n ubermag_env python=3.10
$ conda activate ubermag_env
$ conda install --channel conda-forge ubermag
```
And I received the error message below. How this problem occurs?
```
(ubermag_env) $ conda install --channel conda-forge ubermag
Collecting package metadata (current_repodata.json): failed

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/exceptions.py", line 1124, in __call__
        return func(*args, **kwargs)
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/cli/main.py", line 69, in main_subshell
        exit_code = do_call(args, p)
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/cli/conda_argparse.py", line 91, in do_call
        return getattr(module, func_name)(args, parser)
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/notices/core.py", line 109, in wrapper
        return func(*args, **kwargs)
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/cli/main_install.py", line 20, in execute
        install(args, parser, 'install')
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/cli/install.py", line 264, in install
        unlink_link_transaction = solver.solve_for_transaction(
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/core/solve.py", line 132, in solve_for_transaction
        unlink_precs, link_precs = self.solve_for_diff(update_modifier, deps_modifier,
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/core/solve.py", line 175, in solve_for_diff
        final_precs = self.solve_final_state(update_modifier, deps_modifier, prune, ignore_pinned,
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/core/solve.py", line 280, in solve_final_state
        ssc = self._collect_all_metadata(ssc)
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/common/io.py", line 84, in decorated
        return f(*args, **kwds)
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/core/solve.py", line 447, in _collect_all_metadata
        index, r = self._prepare(prepared_specs)
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/core/solve.py", line 1060, in _prepare
        reduced_index = get_reduced_index(self.prefix, self.channels,
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/core/index.py", line 267, in get_reduced_index
        new_records = SubdirData.query_all(spec, channels=channels, subdirs=subdirs,
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/core/subdir_data.py", line 124, in query_all
        result = tuple(chain.from_iterable(executor.map(subdir_query, channel_urls)))
      File "/home/USER/miniconda3/lib/python3.10/concurrent/futures/_base.py", line 621, in result_iterator
        yield _result_or_cancel(fs.pop())
      File "/home/USER/miniconda3/lib/python3.10/concurrent/futures/_base.py", line 319, in _result_or_cancel
        return fut.result(timeout)
      File "/home/USER/miniconda3/lib/python3.10/concurrent/futures/_base.py", line 458, in result
        return self.__get_result()
      File "/home/USER/miniconda3/lib/python3.10/concurrent/futures/_base.py", line 403, in __get_result
        raise self._exception
      File "/home/USER/miniconda3/lib/python3.10/concurrent/futures/thread.py", line 58, in run
        result = self.fn(*self.args, **self.kwargs)
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/core/subdir_data.py", line 113, in <lambda>
        subdir_query = lambda url: tuple(
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/core/subdir_data.py", line 129, in query
        self.load()
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/core/subdir_data.py", line 215, in load
        _internal_state = self._load()
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/core/subdir_data.py", line 342, in _load
        _internal_state = self._read_local_repodata(mod_etag_headers)
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/core/subdir_data.py", line 399, in _read_local_repodata
        _internal_state = self._process_raw_repodata_str(
      File "/home/USER/miniconda3/lib/python3.10/site-packages/conda/core/subdir_data.py", line 456, in _process_raw_repodata_str
        json_obj = json.loads(raw_repodata_str or "{}")
      File "/home/USER/miniconda3/lib/python3.10/json/__init__.py", line 346, in loads
        return _default_decoder.decode(s)
      File "/home/USER/miniconda3/lib/python3.10/json/decoder.py", line 337, in decode
        obj, end = self.raw_decode(s, idx=_w(s, 0).end())
      File "/home/USER/miniconda3/lib/python3.10/json/decoder.py", line 355, in raw_decode
        raise JSONDecodeError("Expecting value", s, err.value) from None
    json.decoder.JSONDecodeError: Expecting value: line 414463 column 5 (char 12941763)

`$ /home/USER/miniconda3/bin/conda install --channel conda-forge ubermag`

  environment variables:
                 CIO_TEST=<not set>
        CONDA_DEFAULT_ENV=ubermag_env
                CONDA_EXE=/home/USER/miniconda3/bin/conda
             CONDA_PREFIX=/home/USER/miniconda3/envs/ubermag_env
           CONDA_PREFIX_1=/home/USER/miniconda3
    CONDA_PROMPT_MODIFIER=(ubermag_env)
         CONDA_PYTHON_EXE=/home/USER/miniconda3/bin/python
               CONDA_ROOT=/home/USER/miniconda3
              CONDA_SHLVL=2
           CURL_CA_BUNDLE=<not set>
          LD_LIBRARY_PATH=:/home/USER/opt/szip-2.1.1/lib:/home/USER/opt/hdf5-
                          1.14.0/build/lib:/home/USER/opt/wannier90-3.1.0/w90/serial/lib
               LD_PRELOAD=<not set>
              LD_RUN_PATH=:/home/USER/opt/szip-2.1.1/lib:/home/USER/opt/hdf5-
                          1.14.0/build/lib:/home/USER/opt/wannier90-3.1.0/w90/serial/lib
                  MANPATH=:
               MODULEPATH=/etc/environment-modules/modules:/usr/share/modules/versions:/usr/shar
                          e/modules/$MODULE_VERSION/modulefiles:/usr/share/modules/modulefiles
                     PATH=/home/USER/bin:/home/USER/opt/vaspkit.1.3.5/bin:/home/USER/minicond
                          a3/envs/ubermag_env/bin:/home/USER/miniconda3/condabin:/usr/local/sbi
                          n:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/g
                          ames:/snap/bin:/home/USER/opt/MATLAB/R2022b/bin:/home/USER/opt/pycha
                          rm-community-
                          2022.3.3/bin:/home/USER/opt/eclipse:/home/USER/p4vasp/bin:/home/qili
                          n/p4vasp/bin
          PKG_CONFIG_PATH=:/home/USER/opt/zlib-1.2.13/lib/pkgconfig:/home/USER/opt/dftd4-
                          3.5.0/install/lib/pkgconfig
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>

     active environment : ubermag_env
    active env location : /home/USER/miniconda3/envs/ubermag_env
            shell level : 2
       user config file : /home/USER/.condarc
 populated config files : /home/USER/.condarc
          conda version : 23.1.0
    conda-build version : not installed
         python version : 3.10.9.final.0
       virtual packages : __archspec=1=x86_64
                          __glibc=2.35=0
                          __linux=5.19.0=0
                          __unix=0=0
       base environment : /home/USER/miniconda3  (writable)
      conda av data dir : /home/USER/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge/linux-64
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge/noarch
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main/linux-64
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main/noarch
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/r/linux-64
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/r/noarch
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/msys2/linux-64
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/msys2/noarch
          package cache : /home/USER/miniconda3/pkgs
                          /home/USER/.conda/pkgs
       envs directories : /home/USER/miniconda3/envs
                          /home/USER/.conda/envs
               platform : linux-64
             user-agent : conda/23.1.0 requests/2.28.1 CPython/3.10.9 Linux/5.19.0-38-generic ubuntu/22.04.2 glibc/2.35
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.

If submitted, this report will be used by core maintainers to improve
future releases of conda.
Would you like conda to send this report to the core maintainers? [y/N]: y

Upload successful.

Thank you for helping to improve conda.
Opt-in to always sending reports (and not see this message again)
by running

    $ conda config --set report_errors true
```

**Comment by lang-m** [comment]
Hi @KylinGuo, that seems to be a `conda` problem, not something caused by Ubermag.

I just tried running the installation locally and it at all worked fine with `conda 23.3.1`. You could try upgrading your conda version from 23.1.0 to 23.3.1.

**Comment by KylinGuo** [comment]
> Hi @KylinGuo, that seems to be a `conda` problem, not something caused by Ubermag.
> 
> I just tried running the installation locally and it at all worked fine with `conda 23.3.1`. You could try upgrading your conda version from 23.1.0 to 23.3.1.

Thank you, @lang-m ! The problem is solved after updating my conda version to 23.3.1.
